src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2046 @@
+
+The newly introduced endpoints consist of:
+* `POST /api/tracker`
+* `GET /api/tracker/enrollments`
+* `GET /api/tracker/events`
+* `GET /api/tracker/trackedEntities`
+* `GET /api/tracker/relationships`
+
+Significant changes occurred in version 2.36 to make the interface with clients homogeneous and to allow more consistent and flexible ways to use the services.
+
+> ***NOTE***
+>
+> The old endpoints are marked as deprecated but still work as before.<br>
+> Some functionality is not yet ready in the new endpoints, but they support their primary use-cases.
+
+
+> ***NOTE 2***
+>
+> These endpoints currently only support the `JSON` format as input/output.<br>
+> Support for the `CSV` format will also be available in the future.
+
+## Changes in the API
+
+Property names used in the API have changed to use consistent naming across all the new endpoints.
+
+### Tracker Import changelog (`POST`)
+
+The following table highlights the differences between the previous tracker import endpoints (version 2.35) and the new endpoint (introduced in version 2.36). All endpoints are still currently available.
+
+|Tracker Object|Previously|Currently|
+|---|---|---|
+|**Attribute**|`created`<br>`lastUpdated`|`createdAt`<br>`updatedAt`|
+|**DataValue**|`created`<br>`lastUpdated`|`createdAt`<br>`updatedAt`|
+|**Enrollment**|`created`<br>`createdAtClient`<br>`lastUpdated`<br>`lastUpdatedAtClient`<br>`trackedEntityInstance`<br>`enrollmentDate`<br>`incidentDate`<br>`completedDate`|`createdAt`<br>`createdAtClient`<br>`updatedAt`<br>`updatedAtClient`<br>`trackedEntity`<br>`enrolledAt`<br>`occurredAt`<br>`completedAt`|
+|**Event**|`trackedEntityInstance`<br>`eventDate`<br>`dueDate`<br>`created`<br>`createdAtClient`<br>`lastUpdated`<br>`lastUpdatedAtClient`<br>`completedDate`|`trackedEntity`<br>`occurredAt`<br>`scheduledAt`<br>`createdAt`<br>`createdAtClient`<br>`updatedAt`<br>`updatedAtClient`<br>`completedAt`|
+|**Note**|`storedDate`<br>`lastUpdated`|`storedAt`<br>`updatedAt`|
+|**ProgramOwner**|`ownerOrgUnit`<br>`trackedEntityInstance`|`orgUnit`<br>`trackedEntity`|
+|**RelationshipItem**|`trackedEntityInstance.trackedEntityInstance`<br>`enrollment.enrollment`<br>`event.event`|`trackedEntity`<br>`enrollment`<br>`event`|
+|**Relationship**|`created`<br>`lastUpdated`|`createdAt`<br>`updatedAt`|
+|**TrackedEntity**|`trackedEntityInstance`<br>`created`<br>`createdAtClient`<br>`lastUpdated`<br>`lastUpdatedAtClient`|`trackedEntity`<br>`createdAt`<br>`createdAtClient`<br>`updatedAt`<br>`updatedAtClient`|
+
+### Tracker Export changelog (`GET`)
+
+The `GET` endpoints all conform to the same naming conventions reported in the previous paragraph. Additionally, we made some changes in regards to the request parameters to respect the same naming conventions here as well.
+
+These tables highlight the version 2.36 differences in request parameters for `GET` endpoints compared to version 2.35.
+
+#### Request parameter changes for `GET /api/tracker/enrollments`
+|V2.35|v2.36|
```

Comment:
same comment about versions. Can we do the comparison between /enrollments and /tracker/enrollments instead?